__tests__/reactTags.test.js

```diff
@@ -639,3 +641,54 @@ test('should allow duplicate tags when allowUnique is false', () => {
     },
   ]);
 });
+
+describe('Test inputFieldPosition', () => {
+  test('should display input field and tags inline when "inputFieldPosition" is inline', () => {
+    const $el = mount(
+      mockItem({
+        inputFieldPosition: INPUT_FIELD_POSITIONS.INLINE,
+      })
+    );
+
+    const $tagContainer = $el.find('.ReactTags__selected');
+    const childLength = $tagContainer.children().length;
+    expect($tagContainer.children().get(childLength-1).props.className).to.equal('ReactTags__tagInput');
+  });
+
+  test('should display input field above tags when "inputFieldPosition" is top', () => {
+    const $el = mount(
+      mockItem({
+        inputFieldPosition: INPUT_FIELD_POSITIONS.TOP,
+      })
+    );
+
+    const $tagContainer = $el.find('.ReactTags__tags');
+    expect($tagContainer.children().get(0).props.className).to.equal('ReactTags__tagInput');
+  });
+
+  test('should display input field below tags when "inputFieldPosition" is bottom', () => {
+    const $el = mount(
+      mockItem({
+        inputFieldPosition: INPUT_FIELD_POSITIONS.BOTTOM,
+      })
+    );
+
+    const $tagContainer = $el.find('.ReactTags__tags');
+    expect($tagContainer.children().get(1).props.className).to.equal('ReactTags__tagInput');
+  });
+
+  test('should show console warning when "inline" is false', () => {
+    const $consoleStub = stub(console, 'warn');
+
+    mount(
+      mockItem({
+        inline: false,
+      })
+    );
+
+    expect( $consoleStub.calledOnce ).to.be.true;
+    expect( $consoleStub.calledWith('[Deprecation] The inline attribute is deprecated and will be removed in v7.x.x, please use inputFieldPosition instead.') ).to.be.true;
```

Comment:
`consoleWarnStub.to.be.calledWithExactly('[Deprecation] The inline attribute is deprecated and will be removed in v7.x.x, please use inputFieldPosition instead.')`

Reply:
Done